model/db/heart_disease_bycounty_ddl.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE "State_symbol" (
+	"StateName"	TEXT,
+	"Symbol"	TEXT,
+	PRIMARY KEY("Symbol")
+);
+
+CREATE TABLE "Heart_Disease_Mortality_County" (
+	"State"	TEXT,
+	"County"	TEXT,
+	"Rate_100000"	NUMERIC,
+	"Rate_Level"	INTEGER,
+	PRIMARY KEY("State","County")
+);
+
+CREATE TABLE "Population_State_County" (
```

Comment:
Rename "State_Counties_Population"

Reply:
I am leaving the main purpose in the first part of the name to signify the table's importance. Here in this case this table is for showing the Population by State and County